lib/SystemStatistics.php

```diff
@@ -94,4 +95,25 @@ protected function getMemoryUsage() {
 		];
 	}
 
+	/**
+	 * Get current CPU load average
+	 *
+	 * @return array load average with three values, 1/5/15 minutes average.
+	 */
+	protected function getProcessorUsage() {
+		// get current system load average.
+		$loadavg = sys_getloadavg();
+
+		// check if we got any values back.
+		if (!(is_array($loadavg) && count($loadavg) === 3)) {
+			// either no array or too few array keys.
+			// returning back zeroes to prevent any errors on JS side.
+			$loadavg = [0, 0, 0];
```

Comment:
Done! Similar as when failing to retrieve memory usage.
![image](https://user-images.githubusercontent.com/2029878/26982450-befcea9a-4d38-11e7-8aed-6f650d2b4ede.png)